src/Hilke.KineticConvolution/AlgebraicNumberCalculatorExtensions.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+
+namespace Hilke.KineticConvolution
+{
+    public static class AlgebraicNumberCalculatorExtensions
+    {
+        public static bool IsZero<TAlgebraicNumber>(
+            this IAlgebraicNumberCalculator<TAlgebraicNumber> calculator,
+            TAlgebraicNumber number)
+        {
+            if (calculator == null)
```

Comment:
```suggestion
            if (calculator is null)
```

Corriger également aux autres endroits.

Reply:
done partout